### ISSUE TITLE
Experiment was added but there is no trace of it

### ISSUE THREAD

**Issue by denkv** [hobbit-project/platform#472]
https://master.project-hobbit.eu/experiments/1570873670213

This experiment was configured and added, but it's not in the queue, not running and not displayed as cancelled, errored or done.

See also https://github.com/hobbit-project/platform/issues/452

**Comment by denkv** [comment]
This can happen when there are connectivity problems and controller cannot store experiment results.

Possible solutions would be to actually do something (what?) when the `INSERT` query fails:
https://github.com/hobbit-project/platform/blob/master/platform-controller/src/main/java/org/hobbit/controller/ExperimentManager.java#L429-L439

**Comment by MichaelRoeder** [comment]
The same problem occurs when the platform can not store the model because Virtuoso does not like the data (e.g., because of `NaN` in a double value).

dump the data in a file might be a good idea. However, it should also log the problem so that we are aware of it.

**Comment by denkv** [comment]
Dumping the experiment result models to files (and having them accessible with HTTP/FTP) sounds like a good idea in general.

**Comment by MichaelRoeder** [comment]
I agree in general. However, I see an issue there: at the moment, we assume that we have a master node that handles the management / communication and a second node for data. Storing the files on the data node and offering them via HTTP (e.g., with nginx or something similar) should be easy. However, the following lines can create issues:

> This can happen when there are connectivity problems and controller cannot store experiment results

If the connectivity between the master and the data node is the issue, both storage components (the one triple store as well as the file writer) wouldn't be reachable.

Hence, the files would have to be stored on the master node which kind of breaks our overall idea of separating the data and the management :thinking:
